blazor/spreadsheet/cellrange.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+title: Cell Range Management in Blazor Spreadsheet | Syncfusion
+description: Checkout and learn here about the cell range including cell formatting, autofill, and clear functionalities in the Syncfusion Blazor Spreadsheet component and more.
+platform: Blazor
+control: Spreadsheet
+documentation: ug
+---
+
+# Cell range in Blazor Spreadsheet component
+
+A group of cells in a sheet is known as cell range.
+
+## Cell Formatting
+
+Cell formatting allows you to visually emphasize data in the Spreadsheet by applying various styles and text formatting to cells or ranges of cells.
+
+Cell formatting options include:
+
+* **Bold** - Emphasizes text with a heavier weight to make it stand out in the Spreadsheet.
+
+* **Italic** - Slants text to create a distinct visual appearance, often used for emphasis or differentiation.
+
+* **Underline** - Adds a line beneath text for highlighting or indicating links.
+
+* **Strikethrough** - Draws a line through text, often used to indicate completed tasks or obsolete data.
+
+* **Font Family** - Changes the typeface of text (such as Arial, Calibri, Times New Roman) to improve readability or aesthetic appeal.
```

Comment:
Changes the typeface of the text (e.g., Arial, Calibri, Times New Roman) to enhance readability or visual appeal.